alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,47 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, in either the platform Prometheus or a user-defined Prometheus
+instance.
+
+## Impact
+
+Prometheus samples in storage might be missing or inaccurate.
+
+## Diagnosis
+
+1. Determine if the alert has been triggered for the instance of Prometheusi
```

Comment:
```suggestion
1. Determine if the alert has been triggered for the instance of Prometheus
```

Reply:
That's what you get by editing with vim now and then xD